package training.practice.exception;

public class SampleUtil {
	public void doSomething() throws Exception {
		throw new Exception("SampleUtil에서 에러가 발생했습니다.");
	}
}
